src/utils/roundFigure.test.js

```diff
@@ -0,0 +1,9 @@
+import roundFigure from './roundFigure';
+
+test('round 1.21 to to first decimal place is equal 1.2', () => {
```

Comment:
to to?

Reply:
https://github.com/SorokinTim/covid-dashboard-rss/pull/20/commits/fc491764e8e952b1a98825af6700d6dae465673a